q3tiledaligner/src/au/edu/qimr/tiledaligner/util/TiledAlignerUtil.java

```diff
@@ -1336,7 +1336,7 @@ public static boolean doesSequenceHaveMostlySingleBaseRepeats(String sequence) {
 		return nonOverlappingRecs;
 	}
 
-	private static Map<ChrPosition, LongRange> loadReferenceIndexMap(String refFile, String refIndexFile) {
+	public static Map<ChrPosition, LongRange> loadReferenceIndexMap(String refFile, String refIndexFile) {
```

Comment:
It seems refIndexFile is not used in this function,  should it change to public static Map<ChrPosition, LongRange> loadReferenceIndexMap(String refFile) ?

Reply:
`refIndexFile` is used (near the end of the method).
Access changed from public to package protected